[FEATURE] Allow specifying project builder as dependency in template packages (#78)
CPS-IT/project-builder: docs/configuration.md

1. Either register it on [Packagist](https://packagist.org/),
2. Use any other Composer registry (e.g. self-hosted [Satis][4] instance),
   or
3. Host the project template on a VCS repository (such as GitHub) to make
   your package available to the project builder.

Once you use the project builder to create a new project, you can
select the appropriate provider that hosts your template package.

> :bulb: You can add the `cpsit/project-builder` package as dependency to
> your template package and Composer will correctly resolve the constraint.
> This way you can define which versions of the project builder are actually
> supported by your package:

Reviewer: ```suggestion
:bulb: You can add the `cpsit/project-builder` package as a dependency to
your template package. Composer will correctly resolve the constraint.
This way, you can define the versions of the project builder actually
supported by your package:
```
